cloudbuild_public.yaml

```diff
@@ -0,0 +1,70 @@
+steps:
```

Comment:
@BYK I did not create a cloud builder trigger for this. Any issue if I go ahead and do that ?

Reply:
I think you can add a trigger for pull requests to test it here?

Reply:
Yes, that I can do. My issue is in connecting the repo. gcb does not have access to this repo yet. When I try to connect it I see there are 14 repos connected (`Selected 14 repositories.`) but If I try to add cdc I suspect that would remove all the sentry repos I am not admin of. So I need to figure this one first.

Reply:
It should not remove the other repos. I went ahead and requested installation of the GCB app to this repo. I think you can approve it as the repo admin. You should also be adding @getsentry/releases as an admin to this repo for release automation, which will give me admin access and then I can take care of it.